src/main/java/seedu/address/model/job/Job.java

```diff
@@ -21,9 +21,7 @@ public class Job {
     private final Set<Tag> requirements = new HashSet<>();
 
     /**
-     * TODO: Implement data validation
-     * TODO: What is meant by this ah ^^?
-     * Every field has no validation for now.
+     * Every field must be present and not null.
```

Comment:
Actually this change was done in a different branch, no idea why it popped up here.